pkg/apis/garden/v1beta1/defaults.go

```diff
@@ -103,6 +103,18 @@ func SetDefaults_Shoot(obj *Shoot) {
 		}
 	}
 
+	if cloud.Packet != nil {
+		if cloud.Packet.Networks.Pods == nil {
+			obj.Spec.Cloud.Packet.Networks.Pods = &defaultPodCIDR
+		}
+		if cloud.Packet.Networks.Services == nil {
+			obj.Spec.Cloud.Packet.Networks.Services = &defaultServiceCIDR
+		}
+		if cloud.Packet.Networks.Nodes == nil && len(cloud.Packet.Networks.Workers) > 0 {
+			obj.Spec.Cloud.Packet.Networks.Nodes = &cloud.Packet.Networks.Workers[0]
```

Comment:
it seems that the worker network can be configured in the `packet.networks` section, so let's put it to the shoot example

Reply:
I don't understand. The worker network **cannot** be configured at all. When you start a packet project, you are given a cidr for your worker nodes and you live with it. There have been discussions about real (configurable) VPC, but nothing in the works for now.

Reply:
OK, then remove it from the API (`PacketNetworks` struct contains `Workers` field)

Reply:
OK, will do.

Reply:
Done.